src/pages/index.js

```diff
@@ -13,26 +15,25 @@ import {
   profileEditModal
 } from "../utils/constants.js";
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a5172d76-7206-4bb6-b609-2e8e17b83724",
+    "Content-Type": "application/json",
+  },
+});
+
 function handleImageClick(cardData) {
   popupWithImage.open(cardData);
 }
 
-function createCard(item) {
-  const cardElement = new Card(item, "#card-template", handleImageClick);
-  return cardElement.getView()
+function renderCard(item) {
+  const card = new Card(item, "#card-template", handleImageClick, handleDeleteButton, handleLikeClick);
+  return card.getView()
 }
 
-function renderCard(cardData) {
-  const card = createCard(cardData);
-  section.addItem(card);
-}
 
-function handleAddCardFormSubmit(inputValues) {
-  renderCard(inputValues);
-  variable.addCardForm.reset();
-  cardFormFormValidator.changeButtonState();
-  cardPopup.close();
-}
+
 
 //FormValidator
```

Comment:
Please, delete `this._disableButton();` from line 70 in `FormValidator.js` because you need to disable the button only in `then` or when you open the form. If there is an error during the request the user should be able to click on the button to submit the form for the 2nd time